finetuner/experiment.py

```diff
@@ -165,14 +167,24 @@ def create_run(
             **kwargs,
         )
 
-        cpu = kwargs.get(CPU, True)
+        device = kwargs.get(DEVICE, 'cuda')
+        if device == 'cpu' and not kwargs.get(CPU, True):
```

Comment:
can you explain the logic here?

Reply:
so it should be `if device == 'cuda' and kwargs.get(CPU, True):`?

Reply:
yes it's more clear